dataactbroker/fileRoutes.py

```diff
@@ -162,36 +162,52 @@ def check_current_page(submission):
         submission_id = submission.submission_id
 
         # /v1/reviewData/
+        # Checks that both E and F files are finished
         review_data = sess.query(Job).filter(Job.submission_id == submission_id,
                                              Job.file_type_id.in_([6, 7]), Job.job_status_id == 4)
```

Comment:
So what happens here if they re-upload A/B/C files and some validations fail? Or cross-file validations fail? We don't change the status of E/F generation if something changes I think. We need to also check that cross-file is done and there are no errors in it.